lib/core/src/display/scene.rs

```diff
@@ -176,7 +179,11 @@ impl {
         let composer = RenderComposer::new(&pipeline,&context,&variables,width,height);
 
         Self {pipeline,composer,root,canvas,context,symbols,camera,symbols_dirty,shape,shape_dirty,logger
-             ,listeners,variables,on_resize,text_components,stats}
+             ,listeners,variables,on_resize,text_components,stats,zoom_uniform}
+    }
+
+    pub fn canvas(&self) -> web_sys::HtmlCanvasElement {
+        self.canvas.clone()
```

Comment:
use `clone_ref` to indicate that it is a cheap clone.

Reply:
There is no `clone_ref` in `HtmlCanvasElement`, but I am changing it everywhere else when possible